exe/facter.cc

```diff
@@ -116,6 +116,15 @@ void log_queries(set<string> const& queries)
     log(level::info, "requested queries: %1%.", output.str());
 }
 
+void print_fact_groups(map<string, vector<string>> const& fact_groups) {
+    for (auto group : fact_groups) {
```

Comment:
I'd suggest using refs, as in `for (auto& group : fact_groups)`. Auto uses value by default unless it's not copyable.